easybuild/tools/github.py

```diff
@@ -1020,6 +1048,20 @@ def merge_pr(pr):
                              pr, pr_target_account, pr_target_repo, status, reviews_data)
     pr_data['reviews'] = reviews_data
 
+    # check for required PRs mentioned in the body
+    match = re.search('[Rr]equires .*', pr_data['body'])
+    if match:
+        pr_deps = {}
+        pr_deps['framework'] = re.findall('(?:framework/pull/)(\d+)', match.group())
+        pr_deps['easyblocks'] = re.findall('(?:blocks/pull/)(\d+)', match.group())
+        pr_deps['easyconfigs'] = re.findall('(?: #|~~#)(\d+)', match.group())
```

Comment:
@migueldiascosta One thing you seem to be overlooking here is that `--merge-pr` also works on framework and easyblock PRs (it follows `--pr-target-repo`).

So, if framework PR is being merged, then we should also take `#[0-9]+` into account for `framework`, etc...

Also, are you taking into account that there may be multiple `requires` entries? That's not entirely clear to me at first glance...

Reply:
@boegel indeed, I was only thinking of easyconfig PRs, will need to check the `pr-target-repo` and update de regexs accordingly

regarding multiple entries, it should find all (in the body) following the first `requires`, but I'll check